aldryn_newsblog/views.py

```diff
@@ -262,7 +263,7 @@ class ArticleSearchResultsList(ArticleListBase):
     def get(self, request, *args, **kwargs):
         self.query = request.GET.get('q')
         self.max_articles = request.GET.get('max_articles', 0)
-        self.edit_mode = (request.toolbar and request.toolbar.edit_mode)
+        self.edit_mode = (request.toolbar and toolbar_edit_mode_active(request))
```

Comment:
Does this also still work with django CMS 3.4, I thought thats 3.6+ only?

Reply:
yes, here is compact: https://github.com/divio/aldryn-newsblog/blob/master/aldryn_newsblog/compat.py#L1

Reply:
Thanks, didn't dig that far 👍